include/znc/Modules.h

```diff
@@ -1050,6 +1050,13 @@ class CModule {
     /// @deprecated Use OnSendToIRCMessage() instead.
     virtual EModRet OnSendToIRC(CString& sLine);
 
+   /** Called immediately before ZNC sends a *status message to the client.
+     *  @since 1.8.0
+     *  @param Message The status message being sent to the client.
+     *  @return See CModule::EModRet.
+     */
+    virtual EModRet OnPutModule(const CString& sModule, const CString& sLine, CClient* pClient);
```

Comment:
If adding a new module callback, need to add it to modperl/modpython too (in the functions.in file)